src/lib/components/chip/QChip.svelte

```diff
@@ -98,6 +98,7 @@
   {...Q.classes}
   aria-disabled={disabled || undefined}
   {tabindex}
+  role="button"
```

Comment:
Should it always have this?

Reply:
Not sure, have to read about this in official ally docs. ImportaProbably not super important atm